frontend/src/lib/components/LemonButton/LemonButton.scss

```diff
@@ -229,7 +229,7 @@
         &.LemonButton--status-primary {
             &:hover {
                 // NOTE: this is a rare case so we are hardcoding the value. Only primary status is supported
```

Comment:
I suppose we aren't hardcoding it anymore… 😅

Reply:
True. I guess the comment was more to say that is is a weirdly specific override but I will update it to say that :D